mcux/mcux-sdk/devices/MIMX9352/drivers/fsl_clock.c

```diff
@@ -16,7 +16,25 @@
 
 static CCM_Type *ccm_base = CCM_CTRL;
 
-volatile uint32_t g_clockSourceFreq[kCLOCK_Ext + 1];
+volatile uint32_t g_clockSourceFreq[kCLOCK_Ext + 1] = {
```

Comment:
This change should also be submitted to the MCUX SDK github repo. Please submit PR here:
https://github.com/nxp-mcuxpresso/mcux-sdk/

Reply:
Already merged. See [this commit](https://github.com/nxp-mcuxpresso/mcux-sdk/commit/bb8b34de85998f5f697f08472fc4655ab6a9f69e).